### PR TITLE
- Add support to oracle SQLcl.

### PR DESCRIPTION
**Pull request by sfc-gh-aramirezfuentes**
- Update the readme.md in the oracle technology.
- Add new file in order to support the extraction-script in Oracle using SQLcl technology.

### CODE REVIEW

File: Oracle/README.md [hunk 1]
```diff
@@ -5,15 +5,63 @@ so it can be migrated to [Snowflake](https://www.snowflake.com/) using [SnowConv
...
+4. After modifying, the `create_ddls.sh` file can be run from the command line to execute the extract. 
+   In order to run the script you can open a terminal in the  `create_ddls.sh` path  and execute the command:
+   ```bash
+   source create_ddls.sh
+   ```
+   or
+   ``` bash
+   sh create_ddls_sqlcl.sh
+   ``` 
+   The following files will be created in the directory `/<SCRIPT_PATH>/object_extracts/DDL`.
```
Comment (sfc-gh-dperezarroyo) [inline Oracle/README.md:60]: Are these all the files that will be created? DDL is the name of the folder, right? but I think the files generated are something like TAbles, Views...

Comment (sfc-gh-dperezarroyo) [inline Oracle/README.md:60]: Can we also mention which files are generated? tables, views... And a related question , does the creation of the file Procedures depends if there is at least one procedure? or is the file always created (empty) if not procedures are found

File: Oracle/README.md [hunk 2]
```diff
@@ -107,6 +171,30 @@ These files will contain the definitions of the objects specified by the file na
 
 You can then zip the `/object_extracts/DDL` so it these files can then be processed with [SnowConvert](https://docs.snowconvert.com/snowconvert/for-oracle/introduction).
 
+## Know errors and FAQs
```
Comment (sfc-gh-dperezarroyo) [inline Oracle/README.md:174]:
```suggestion
## Known errors and FAQs

```

File: Oracle/README.md (hunk 1 above)
Comment (sfc-gh-aramirezfuentes) [reply Oracle/README.md:60]: the files are always generated by the tool. about the files that are generated you can see the description in the DDL Files. I added this comment in the documentation

**Pull request closed without merging**